### DISCUSSION TITLE
Attribute Error

### DISCUSSION THREAD

**Discussion by gvasilakop** [MaartenGr/BERTopic#749] [Q&A] [answered]
Hello,

For starters thank you for your amazing work.

I am using BERTopic to get some topics out and it gives me 28 topics (using nr_topics='auto' and min_topic_size = 300).

After observing word scores i found a lot of simillarities in the topics generated. Instead of reducing them i tried to merge them but i get the following:

**AttributeError: 'BERTopic' object has no attribute 'merge_topics'**

Can you please guide me on how to resolve this? Is it because i used "auto"?

Thanks

**Comment by MaartenGr** [comment, marked as answer]
This feature was introduced in one of the newer versions of BERTopic. Are you using the latest version of the package?
